python/pylibcudf/pylibcudf/tests/conftest.py

```diff
@@ -151,6 +140,32 @@ def _generate_nested_data(typ):
     ), pa_table
 
 
+@pytest.fixture(scope="session", params=[0, 100])
+def table_data(request):
+    """
+    Returns (TableWithMetadata, pa_table).
+
+    This is the default fixture you should be using for testing
+    pylibcudf I/O writers.
```

Comment:
Should we name the fixture accordingly? `table_data` is more generic than `io_table_data` or so.

Reply:
Ill do this in a follow up